src/john.c

```diff
@@ -2015,3 +2006,18 @@ int main(int argc, char **argv)
 
 	return exit_status;
 }
+
+#ifdef HAVE_LIBFUZZER
+uint8_t buffer[8192];
+
+// dummy fuzzing target
+int LLVMFuzzerTestOneInput(const uint8_t *Data, size_t Size)
+{
+	memcpy(buffer, Data, Size);
+	buffer[Size] = 0;
```

Comment:
If "Size" is literally the size, then "buffer[Size]" is out of bounds (off-by-one).